openshift/all-jobs-template.yaml

```diff
@@ -0,0 +1,293 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: solver
+  annotations:
+    description: "Thoth: Solver Job Template"
+    openshift.io/display-name: "Thoth: Solver"
+    version: 0.9.0
+    tags: thoth,ai-stacks,aistacks,solver,machinlearning
+    template.openshift.io/documentation-url: https://github.com/Thoth-Station/
+    template.openshift.io/long-description: >
+      This template defines resources needed to run solvers in Thoth, it is used
+      to create new OpenShift Jobs running the analyzer.
+    template.openshift.io/provider-display-name: "Red Hat, Inc."
+  labels:
+    template: solver
+    app: thoth
+    component: solver
+
+parameters:
+  - description: Registry the ImageStream to be use lives in
+    displayName: ImageStream Registry
+    required: true
+    name: IMAGE_STREAM_REGISTRY
+    value: "docker-registry.default.svc:5000"
+
+  - description: Namespace the ImageStream to be use lives in
+    displayName: ImageStream Project
+    required: true
+    name: IMAGE_STREAM_NAMESPACE
+    value: "thoth-infra-stage"
+
+  - description: Tag of the ImageStream to be use
+    displayName: ImageStream Tag
+    required: true
+    name: IMAGE_STREAM_TAG
+    value: "latest"
+
+  - name: THOTH_SOLVER_NO_TRANSITIVE
+    description: "Do not resolve transitive dependencies"
+    displayName: "No transitive dependencies"
+    required: true
+  - name: THOTH_SOLVER_JOB_ID
+    description: "Identifier of a job run."
+    displayName: "Job identifier"
+    required: true
+  - name: THOTH_SOLVER_PACKAGES
+    description: "Packages to be resolved"
+    displayName: "Packages to be analyzed by solver"
+    required: true
+  - name: THOTH_SOLVER_OUTPUT
+    description: "Output to send solver results to"
+    displayName: "Solver output"
+    required: true
+  - name: THOTH_LOG_SOLVER
+    description: "Log solver actions"
+    displayName: "Log solver actions"
+    required: true
+    value: INFO
+  - name: THOTH_SOLVER_INDEXES
+    description: "A comma separated list of indexes to scan during solver run"
+    displayName: "Python Package Indexes"
+    value: ""
+
+objects:
+  - apiVersion: batch/v1
+    kind: Job
+    metadata:
+      name: "${THOTH_SOLVER_JOB_ID}"
+      labels:
+        app: thoth
+        component: solver
+        solver-type: solver-ubi-8-py36
```

Comment:
We cannot use `ubi` here as we would like to match data coming from inspections. Please rename to `solver-rhel-8.0-py36`. Here and other places as well. Solver `ubi` will not exist, it will be mapped to `rhel`.